packages/web/src/scss/components/Navigation/_NavigationLink.scss

```diff
@@ -0,0 +1,52 @@
+@use '@tokens' as tokens;
+@use '../../tools/typography';
+@use 'theme';
+
+.NavigationLink {
+    @include typography.generate(theme.$link-typography);
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: var(--#{tokens.$css-variable-prefix}header-height);
+    padding-inline: theme.$link-spacing;
+    color: theme.$link-color-state-default;
+    border-top: theme.$link-border-bottom-width theme.$link-border-bottom-style transparent;
```

Comment:
I see what you mean, but it's a bit confusing to see bottom border variables in the top border 🙂. I think a comment could make it clear.

Reply:
I added a comment.